qemu/tests/cfg/hpt_max_page_size.cfg

```
@@ -0,0 +1,23 @@
+- hpt_max_page_size: install setup image_copy unattended_install
+    type = hpt_max_page_size
+    only ppc64 ppc64le
+    virt_test_type = qemu
+    required_qemu = [3.1.0,)
+    kill_vm = yes
+    setup_hugepages = yes
+    machine_type_extra_params = max-cpu-compat=power8
+    variants:
```

I think this will make the variants easier to read and modify
```suggestion
    variants:
        - without_hugepages:
        - with_hugepages:
            setup_hugepages = yes
            extra_params += " -mem-path /mnt/kvm_hugepage"
    variants:
        - 64k:
            machine_type_extra_params += ,cap-hpt-max-page-size=64k
        - 16m:
            only without_hugepages # If correct
            machine_type_extra_params += ,cap-hpt-max-page-size=16M
            variants:
                - tcg_mode:
                    machine_type_extra_params += ,accel=tcg
                    disable_kvm = yes
                    auto_cpu_model = yes
```

---

Yes, I agree thanks.